org/models.py

```diff
@@ -50,3 +54,13 @@ def __str__(self):
 
     class Meta:
         ordering = ["student__user__first_name"]
+
+
+@receiver(post_save, dispatch_uid="rebuild_website")
+def rebuild_website(sender, instance, **kwargs):
+    if sender not in [Member, Role, Team]:
+        return
+    if settings.REBUILD_WEBHOOK_URL is not None:
+        requests.post(settings.REBUILD_WEBHOOK_URL)
+    else:
+        print("website build triggered")
```

Comment:
We shouldn't print on error right? We should log to sentry or something

Reply:
this isn't an error: it's just to make sure that there's *some* side effect whether or not the webhook URL is set. If it is set, then it hits the webhook. If it's not set (in CI or in dev), it'll print to the console.

Reply:
Do we need a side effect though? I don't think it matters in development or CI